packages/php-wasm/progress/src/lib/progress-tracker.ts

```diff
@@ -0,0 +1,184 @@
+const EPSILON = 0.0001;
```

Comment:
see also [`Number.EPSILON`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/EPSILON) for a built-in and accurate value.

JS numbers really do have some convenient methods and properties.

Reply:
Oh yay, thank you!